docs/source/guide/release_notes.md

```diff
@@ -12,10 +12,69 @@ meta_description: Discover what's new and improved, and review bug fixes, in the
+- Pin and create shared view of relevant projects in your workspace: Allow users to [pin or unpin projects](manage_users.html#More-menu-to-Pin-or-Unpin-and-Duplicate-projects) to more easily filter and find projects of interest. This feature allows you to pin and create a shared view of relevant projects in your workspace. Administrators now have the ability to pin and unpin projects for quicker reference by team members, whether they are projects the team is currently working on, or projects of interest. All team members will have a view of pinned projects to collaborate on in their workspace [DEV-2069].
+- Zoom, align, and pixelate images for annotation: Annotators can zoom and expand images they are labeling within the viewport of the task, set the image to be positioned vertically (top, center, bottom) and horizontally (left, center, right) to find the most convenient position. Once an image is zoomed in, they can also choose to pixelate the image if it is blurry for more precise annotations [DEV-2504].
+- Add the ability to sustain the collapse state of the label across tasks [DEV-2755].
+- This release provides consistent ordering of the label groups [DEV-2755].
+- Enforce the change in the rotate anchor of bounding boxes around image boundaries [DEV-2671].
+- Allow annotators and reviewers to filter view of transcriptions by **author name** [DEV-2669].
+- This release improves the polygon point removal when you create a new polygon. In addition to the double-click and alt-click, now you can use the undo hotkeys (`ctrl/cmd+z`) to remove the point you set or redo it by clicking the `ctrl/cmd+shift+z` hotkeys [DEV-2576].
+- Migrate Label Studio Enterprise and Label Studio Frontend to Playwright for end-to-end activity [DEV-2481].
+
+### Bug fixes
+
+Label Studio 2.3 includes the following bug fixes:
+
+- Fixed an issue when the text captured by a region was not displayed in the **Details** panel [DEV-2958].
+- Resized the left panels in **Quick View** mode to work consistently when expanded to the right in **Label** or **Review** stream, and stop after the maximum width was reached [DEV-2952], [DEV-2943].
+- Fixed an issue when grouping by empty label (**Group by Label**) caused the Label Studio App to crash [DEV-2942].
```

Comment:
I see this ticket and other ones refer to the Outliner feature that is not a part of this release. Please remove everything related to the Outliner Epic

Reply:
I deleted entries based on https://heartex.atlassian.net/browse/DEV-1544. @niklub please let me know if there are any further changes in this regard.